### ISSUE TITLE
getWrappedInstance not available with composeWithTracker

### ISSUE THREAD

**Issue by danieljvdm** [kadirahq/react-komposer#100]
I use this following code:

```
export const TransferModalContainer = composeWithTracker(
  composer,
  null,
  null,
  { withRef: true }
)(TransferModal);
```

and when I print out `this.refs.transferModal` in the parent there is no `getWrappedInstance` method available.

**Comment by clayne11** [comment]
Is the `TransferModal` component a class-based React component? Functional components cannot have `refs`.

**Comment by danieljvdm** [comment]
Yes it's class-based. I just refactored that specific class to use Redux
instead of react-komposer and without editing the dumb component at all,
the ref works.

**Comment by Mokto** [comment]
Same issue here.

**Comment by clayne11** [comment]
Could either of you post a reproduction repo? I implemented this feature and it's working fine for me. There's also tests for it as well so I'm not sure how to reproduce this. If you can post a repro I'm happy to take a look at it.

**Comment by Mokto** [comment]
Sure ! 

https://github.com/Mokto/kickstart-mantrajs-webpack

I tried to set withRef on the parent and child containers. 

When I put withRef on the parent container, I get a weird error =>
`Stateless function components cannot be given refs (See ref "wrappedInstance" in DefaultLoadingComponent created by Container(PostList)). Attempts to access this ref will fail.`
But I do not think I have stateless containers...

Thanks :)

**Comment by clayne11** [comment]
In retrospect that makes sense - the [default loading component](https://github.com/kadirahq/react-komposer/blob/master/lib/common_components.js) is a stateless functional component.

That shouldn't stop the `ref` from working on the component you pass in, though. I'll have a look at your repo.

**Comment by gflores** [comment]
@danieljvdm Had the same problem as you but then I updated to v1.13 and now it works !

@Mokto I have the same Warning message as yours but it doesn't prevent it from working well, I can access wrappedInstances just fine.

**Comment by Mokto** [comment]
Hmmm :( 

I still got this issue.

**Comment by clayne11** [comment]
@Mokto I don't even see `react-komposer` in your `package.json`.
